libsolidity/interface/CompilerStack.cpp

```diff
@@ -1256,6 +1256,18 @@ ContractDefinition const& CompilerStack::contractDefinition(std::string const& _
 	return *contract(_contractName).contract;
 }
 
+std::vector<ContractDefinition const*> CompilerStack::contractDefinitions(
+	std::string const& _sourceName
+) const
+{
+	std::vector<ContractDefinition const*> contracts;
+	if (auto source = m_sources.find(_sourceName); source != m_sources.end())
```

Comment:
```suggestion
	if (
		auto const& source = m_sources.find(_sourceName); 
		source != m_sources.end()
	)
```

aside from that, do you think it would make sense to assert that there actually is a contract definition with the source name instead of silently returning an empty vector?

Reply:
That's a good idea :)